packaging/windows/buildpackage.sh

```diff
@@ -85,5 +85,12 @@ if [ $? -eq 0 ]; then
 fi
 popd > /dev/null
 
+echo "Packaging zip archive"
+pushd "${BUILTDIR}" > /dev/null
+find "${SRC_DIR}/thirdparty/download/windows/" -name '*.dll' -exec cp '{}' '.' ';'
+zip "${PACKAGING_INSTALLER_NAME}-${TAG}-win" -r -9 * --quiet --symlinks
+mv "${PACKAGING_INSTALLER_NAME}-${TAG}-win.zip" "${OUTPUTDIR}"
```

Comment:
Can you please call this `-winportable` to make it clearer that this is a "portable" copy rather than an installer?

Reply:
done.